docs/ui/TeaserCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { Card, Stack, Tiles } from '@/ui';
+import type { ReactElement } from 'react';
+
+import Link from 'next/link';
+
+export interface TeaserCardProps {
+  href: string;
+  icon: ReactElement;
+  title: string;
+  caption: string;
+}
+
+export const TeaserCard = ({ href, icon, title, caption }: TeaserCardProps) => (
+  <Link className="flex no-underline" href={href}>
+    <Card variant="hovering">
+      <Stack space={4}>
+        <div className="bg-bg-muted grid size-12 place-items-center rounded-full p-2">
+          {icon}
+        </div>
+        <div className="text-2xl font-semibold tracking-tight">{title}</div>
+        <div className="text-text-primary-muted text-sm">{caption}</div>
+      </Stack>
+    </Card>
+  </Link>
+);
+
+export interface TeaserListProps {
```

Comment:
The file is atm small so it is fine but in general I would suggest to place all sort of type definitions at the top of file 😬  but maybe just my personal gusto

Reply:
I like to have it next to the component is used :D But yeah we can definitely agree on something and add it to our dev docs?